.github/workflows/cron.yml

```diff
@@ -0,0 +1,15 @@
+name: PRs, Pushes, Cron
+
+on:
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
q: is this workflow going to run every 8am or just once at 8am? why add this workflow?

Reply:
every. it has been here. just isolating it. plus we want something running at least once a day for now...